vaadin-testbench-core-junit5/src/main/java/com/vaadin/testbench/BrowserTestBase.java

```diff
@@ -20,6 +22,7 @@
  * A superclass with helper methods to aid TestBench developers create a JUnit 5
  * based tests.
  */
+@Execution(ExecutionMode.CONCURRENT)
```

Comment:
Hope this can be overriden in the test class?

Reply:
Yes, it can be overriden by `@Execution(ExecutionMode.SAME_THREAD)` or same result will be after setting `testInParallel` property value to 1.